Adds link to model card metadata spec (#1171)
huggingface/huggingface_hub: src/huggingface_hub/templates/modelcard_template.md

---

> {{card_data}} # Please use the metadata as defined in https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1

Reviewer: ```suggestion
# For reference on model card metadata, see: https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1
{{card_data}}
```

Reviewer 2: I like @nateraw 's suggestion of having it on a separate line.
PR is fine for me anyway. Fell free to merge when you both agree on it @meg-huggingface @nateraw :slightly_smiling_face:

Author: Yeah, sounds good to me! Committed.
